list_vs_set.py

```diff
@@ -0,0 +1,44 @@
+"""
+This program creates one list and one set from all the words in the file "words.txt"
+then search for a specific words for 1000 times in each of them and print the time
+it took to conduct each search.
+"""
+
+import time
+
+
+def create_list():
+    # returns a list from all the words in the file 'words.txt'
+    file = open('words.txt', 'r')
+    return list(file.read().split())
+
+
+def create_set():
+    # returns a set from all the words in the gile 'words.txt'
+    return set(create_list())
+
+
+def average_runtime():
+    words_list = create_list()
+    words_set = create_set()
+
+    start_time = time.time()
+    for x in range(1000):
+        if 'zwitterion' in words_list:
+            pass
+        else:
+            pass
+    end_time = time.time()
+    print(f'running search in words_list took {end_time - start_time} ')
+
+    start_time = time.time()
+    for x in range(1000):
+        if 'zwitterion' in words_set:
+            pass
+        else:
```

Comment:
Same comments for this part.
Also, why not merging the loops?

Reply:
I dont know how

Reply:
also each loop tkes different time so I cant merge them